lesson6.js

```diff
@@ -19,6 +15,10 @@ function deleteLoadingImg() {
 
 function makeLinkList() {
   makeLoadingImage();
```

Comment:
IMO

I feel `makeLoadingImage` is not a part of `makeLinkList`.
Because the name `makeLinkList` means only making lists.
If by any chance, you reuse this function for making lists in other part, every time `makeLoadingImage` will be called.

I think it would be better to call it before `makeLinkList().then((item)` or rename `makeLinkList` to a suitable name.

Reply:
> rename makeLinkList to a suitable name.

I agree with Yuka-san.
I'm sorry for the confusion, as I'm the one who gave the IMO that "makeLoadingImage" should be in "makeLinkList." In that case, I should have explained from the beginning that the function name needs to be changed. My apologies.

Reply:
I didn't notice the difference between the function name and the processing content. After reading the comments, I noticed the strangeness. thank you for teaching me. Changed from "makeLinkList" to "LoadingForMakeList".

Reply:
As Ms. @443n0511 said, I also think `LoadingForMakeList` sounds a little not fit for this function. 
And we could create a part of making lists as a function. I should have told you about this at the beginning. Sorry about that.

I think the above code is also good. :)
But `fetchListData` means like a "getting list data" so I thought `addLoadingImg` might be a little not fit for this function.

This is just IMO.
When I tried this lesson, I used `window.onload`, and put `fetchListData`, `addLoadingImg` and `info`(this is the variable name from your code.) inside.
It's like this↓
```
window.onload = () => {
  addLoadingImg();
  const fetchListData = new Promise((resolve, reject) => {
    const info = [
      { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
      { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
    ];
    setTimeout(() => {
      resolve(listItems);
    }, 3000);
  });
```
`.onload` executes the script after the web page has been loaded.
About `.onload`(https://techacademy.jp/magazine/15558)

So after loading the web page, this `window.onload` is going to execute. 
After that you can call the `fetchListData().then((item))` part.

Also, I think the variable name `info` is a bit abstract. It would be better if it had a more concrete name, such as `listInfo` or `listItems`:)

Reply:
Thank you for your comment. When I created the function "createList" that creates the list, I found that the name of "LoadingForMakeList" was strange. This means that the list will only be created after the data has been successfully retrieved. This was a great learning experience.

> As for English vocabulary, I referred to this page.

Thank you for the reference URL. I will continue to do my best to name functions.

> When I tried this lesson, I used window.onload, and put fetchListData, addLoadingImg and info(this is the variable name from your code.) inside.

"Window.onload" I learned this for the first time. You can write the process you want to execute when the reading is completed. I learned about other methods as well.

> Also, I think the variable name info is a bit abstract. It would be better if it had a more concrete name, such as listInfo or listItems :)
I learned that if the variable name is abstract, it is difficult to understand the processing of the function.
Thank you for your guidance.